src/lib/sdc/usb-key.sh

```diff
@@ -133,6 +135,45 @@ function unmount_usb_key()
 	umount "$mnt"
 }
 
+# Evil twins of the {,un}mount_usb_key functions.  These are specific
+# to ISO/hsfs (aka. {C,DV,B}D-ROM) disks.  We may be able to factor-out
+# even more common bits from the usb_key functions, but not today.
+
+function mount_ISO
+{
+	local mnt=`extract_mountpath $1`
+
+	# XXX KEBE SAYS JUST TRUST IT FOR NOW...
+	for disk in `disklist -r`; do
+		mount -F hsfs /dev/dsk/${disk}s0 $mnt
```

Comment:
If this fails, will it print out an error?
`fstyp` could be useful here to weed out non-hsfs devices -- wouldn't need to use  `2>/dev/null` to quiet things we just want to skip, but still produce error messages for things we are interested in.

Reply:
Yes.  The function returns 0 (success) mid-loop, and the bottom of the function is the failure case.